trie/stacktrie.go

```diff
@@ -291,9 +305,8 @@ func (t *StackTrie) insert(st *stNode, key, value []byte, prefix []byte) {
 //   - And the 'st.type' will be 'hashedNode' AGAIN
 //
 // This method also sets 'st.type' to hashedNode, and clears 'st.key'.
-func (t *StackTrie) hash(st *stNode, path []byte) {
-	// The switch below sets this to the RLP-encoding of this node.
-	var encodedNode []byte
+func (t *StackTrie) hash(st *stNode, prefix []byte) {
```

Comment:
why `path`->`prefix`? I mean, this is the full path, not just some prefix for something.. ?